new/telemetry.md

```diff
@@ -0,0 +1,136 @@
+---
+RFC: unassigned
+Title: Anonymous user behaviour data collection
+Author: Thom May <thom@chef.io>, Charles Johnson <charles@chef.io>
+Status: Draft
+Type: Informational
+---
+
+# Anonymous user behaviour data collection
+
+In order to provide greater insight into how our Chef users interact with Chef
+tools, this RFC introduces a framework for defining the policies for the
+anonymous collection (sharing), storage, and deletion of product usage data
+that can be applied to any Chef project.
+
+## Scope
+
+This RFC only covers Chef's open source ecosystem. It does not apply to proprietary,
+commercially available tools made by Chef Software.
+
+## Motivation
+
+    As a UX designer at Chef,
+    I want to understand how people use Chef tools,
+    so that I can improve their experience.
+
+    As a product owner at Chef,
+    I want to learn which Chef tools and commands are most used,
+    so I can prioritise development.
+
+## Specification
+
+This RFC defines the core of product usage data sharing. It does not define how
+individual apps will implement product usage data sharing, and it is expected
+that each app would create a follow up RFC that details individual behaviour,
+and the types of data that will be shared with Chef Software.
+
+### Sharing by default
+
+In order to gather meaningful data, Chef tools will share anonymized usage data
+by default. Experience (with Habitat and Automate) has shown that disabling
+sharing by default provides sample sets that are too small (and potentially
+biased) to draw meaningful conclusions.  The Chef tools must provide clear
+information so that users always know how to easily change their data sharing
+preference. Once set, any data sharing preference saved on an individual host
+will be honored by all Chef tools run on that host.
+
+Server tools may each require their own separate data sharing preference.
+
+### High-Level Implementation
+
+All Chef tools will use a centralized configuration for product usage data
+sharing, in order to provide a consistent experience. If the configuration is
+not present, a data-sharing-enabled application will create a well-formatted
+configuration file. Upon creating the configuration file, the tool will notify
+the user that product usage data sharing is enabled, and will provide clear
+information so that users always knowhow to easily change their data sharing
+preference.
+
+All product usage data sharing enabled applications will provide simple ways
+for a user to check their data sharing preference, and to change that
+preference.
+
+### Privacy and Data Retention
+
+To ensure that sensitive data is not collected, Chef tools should never
+collect parameters passed to command-line options, and tools will be able to
+filter more aggressively if required.
+
+For example, knife may report that a `-P` option was invoked, but will never
+collect the password parameter that was passed to that option.
```

Comment:
well, we want to know about people running unsuccessful commands, but maybe in that case we should just collect which command was tried.